composer.json

```
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^8,<8.2|>=8.2.9,<8.3",
+        "php": "^8,<8.2|>=8.2.9,<8.4",
```

thought: it may be better to not place this upper bound on the minor version, maybe we do the major version instead? We keep running into being blocked everytime a new PHP version comes out, may be better if people self regulated the higher end minor to minor.

```suggestion
        "php": "^8,<8.2|>=8.2.9,<9.0",
```

---

I think it would be better to just exclude unsupported versions unless you're going to react faster to requests like this.

---

Yeah I was heavily involved in the linked thread. My main concern is by opening up the upper bound, the community can better troubleshoot and react to future PHP releases instead of needing to wait the long periods of time we have for upstream changes.